style.css

```diff
@@ -0,0 +1,196 @@
+/* CSS Fiile for Davins Portfolio */
+
+
+/* -------------------------------------------------------------
+	Generic Body Stuff 
+---------------------------------------------------------------- */
+* {
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+/*	background-color: rgb(61, 123, 165);*/
+	font-family: Verdana;
+	font-size: 100%;
+}
+
+
+/* ----------------------------------------------------------
+	Generic Classes for Containers & Cursive Font for Sections
+------------------------------------------------------------- */
+/* Need to add some Media Queries here */
+.mainContainerGrid {
+}
+
+.colorBackgroundDark {
+	/*darker stone blue*/
+	background-color:  hsl(204, 46%, 52%);
+	padding: 1px 0px;
+	padding-top: 10px;
+}
+
+.colorBackgroundLight {
+	/*lighter stone blue*/
+	background-color: hsl(204, 46%, 44%);
+	padding: 1px 0px;
+	padding-top: 10px;
+}
+
+.fontCursive {
+	 font-family:'Monotype Corsiva','Apple Chancery','ITC Zapf Chancery','URW Chancery L',cursive;
+}
+
+
+/* ------------------------------------------
+	HEADER & NAV 
+--------------------------------------------- */
+header {
+	text-align: center;
+	font-size: 120%;
+}
+
+.colorHeaderFooter {
+	background-color: rgb(87, 24, 103);
+	color: white;
+	/* Reminder - need to change followed-links color not just font color */
+}
+
+.fontSizeNav {
+	font-size: 200%;
+}
+
+nav {
+	padding: 20px;
+}
+
+nav a {
+	padding: 10px;
+	color: white;
+	text-decoration: none;
+}
+
+nav a:hover {
+	background-color: rgb(61, 123, 165);
+}
+
+
+/* ---------------------------------------------------------------------
+	MAIN Section with background image & containers & content formatting 
+------------------------------------------------------------------------ */
+
+main {
+
+}
+
+.bkgImageCoding {
+/*	background-image: url("bkg_coding_dark.jpg");*/
+	background-image: url("bkg_dav_coding_dark.jpg");
+	width: 100%;
+	height: auto;	
+	background-size: cover; /* user cover to get full width, instead of contain */
+	background-repeat: no-repeat;
+	padding: 20px 10px;
+}
+
+.containerFlex {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	align-content: space-around;
+	width: 100%;
+}
+
+.articleBox {
+	padding: 10px;
+	margin: 10px;
+	flex: 1 1 250px;
+	text-align: center;
+}
+
+.contentIntro {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+	align-items: center;
+	width: 50%;
+	padding-left: 300px;
+}
+
+.textAboutMe {
+	color: white;
+/*	background-color: black;*/
+	background-color: rgb(87, 24, 103);
+	padding: 10px;
+	vertical-align: middle;
```

Comment:
When the webpage is shrunk or changed in size the box with your name and extra information shifts to the right. It then stays there once shrunk.

Reply:
Yes thats fine for now (the idea was to keep the text on the right so it doesn't overlap the code on the left of the coding-screenshot behind it). Can look into this further when I do more responsive design